src/blockchain/contracts/TwoPhaseCommit.sol

```diff
@@ -1,9 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.4.22 <0.9.0;
+pragma experimental ABIEncoderV2;
 
 contract TwoPhaseCommit {
     enum Ballot {
-        UNKNOWN, // Saved for unexpected unset value only.
+        UNKNOWN,
```

Comment:
Does UNKNOWN now are treated as default value?

Reply:
Correct. I just want to avoid using the first field for any valid value because it's translated into 0 in the unit-test. (Solidity doesn't support enum comparison in unit-test yet so people translate them into int.)